collamat_kareen/Activity3/style.css

```diff
@@ -0,0 +1,67 @@
+body{
+	color: #473C38;
+	font-family: Open Sans;
+}
+a{
+	color:#AE7E5E;
+}
+h1, h2, h3{	
+	font-family: Montserrat;	
+}
+h1{
+	font-size: 40px;
+	color: #6B4C3A;
+}
+
+hr{
+	width:30%;
+	color: #AE7E5E;
+}
+.container{
+	padding-bottom: 180px;
+}
+.upperfold{
+	background-color: #F8F7F2;
+	border-bottom: 2px solid #AE7E5E;
+	width: 100%;
+	padding-top: 80px;
+}
+.upperfold-content{
+	height: 100%;
+	margin: auto;
+	width: 1100px;
+}
+img{
```

Comment:
Please do not use global tag css declaration please use class instead

Reply:
Updated, sir.